src/components/IndividualRecipe.js

```diff
@@ -0,0 +1,173 @@
+    const [id] = useState(localStorage.getItem('recipeId'))
+    const [token] = useState(localStorage.getItem('token'))
+
+    useEffect(() => {
+        fetchRecipe(id)
+    }, [])
+
+    const navigate = useNavigate()
+
+    const handleChange = (event) => {
+
+            setCuisineType(event.target.value)
+
+    };
+
+    const handlePatch = (event) => {
+        event.preventDefault()
+        const recipeObject = {
+            recipeName: recipeName || recipe.recipeName,
+            cuisineType: cuisineType || recipe.cuisineType,
+            estTimeOfPrep: estTimeOfPrep || recipe.estTimeOfPrep,
+            ingredients: ingredients || recipe.ingredients,
+            prepInstructions: prepInstructions || recipe.prepInstructions
+        }
+        fetchEdit(id, recipeObject)
+    }
+
+
+    const fetchRecipe = async (id) => {
+
+        const url = `https://recipe-blog-l7ey.onrender.com/recipe-blog/mypage/${id}`
+
+        const options = {
+            method: 'GET',
+            headers: {
+                Authorization:`Bearer ${token}`,
+                'Content-Type': 'application/json'
+            }
+        }
+
+        try {
+            const response = await fetch(url, options)
+            const data = await response.json()
+
+            setRecipe(data.recipe)
+        } catch (error) {
+            console.log(error)
+        }
+    }
+
+    const fetchEdit = async (id, editedRecipe) => {
+
+        const url = `https://recipe-blog-l7ey.onrender.com/recipe-blog/mypage/${id}`
+
+        const options = {
+            method: 'PATCH',
+            headers: {
+                Authorization:`Bearer ${token}`,
+                'Content-Type': 'application/json'
+            },
+            body: JSON.stringify({
+                "recipeName": editedRecipe.recipeName,
+                "cuisineType": editedRecipe.cuisineType,
+                "estTimeOfPrep": editedRecipe.estTimeOfPrep,
+                "ingredients": editedRecipe.ingredients,
+                "prepInstructions": editedRecipe.prepInstructions
+            })
+        }
+
+        try {
+            const response = await fetch(url, options)
+            const data = await response.json()
+
+            setRecipe(data.updatedRecipe)
+            setEdit(false)
+        } catch (error) {
+            console.log(error)
+        }
+    }
+
+    const fetchDelete = async (id) => {
+        const url = `https://recipe-blog-l7ey.onrender.com/recipe-blog/mypage/${id}`
+
+        const options = {
+            method: 'DELETE',
+            headers: {
+                Authorization:`Bearer ${token}`,
+            }
+        }
+
+        try {
+            const response = await fetch(url, options)
+
+            if (!response.ok) {
+                throw new Error(`Error: ${response.status}`)
+            }
+
+            navigate('/recipe-blog/mypage')
+        } catch (error) {
+            console.log(error)
+        }
+    }
+
+    
+    return (
+        <>  {
```

Comment:
Add the following code to display an error message if the error state is not null:
```
{errorMessage !== null && (
  <div>
    <h3 style={{textAlign: 'center', color: '#F00' }}>{errorMessage}</h3>
  </div>
)}
```